src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,122 @@
+import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { GiHamburgerMenu } from 'react-icons/gi';
+import { VscChromeClose } from 'react-icons/vsc';
+import { SiTailwindcss } from 'react-icons/si';
+
+function Navbar() {
+  const [dark, setDark] = useState(false);
+  const [burger, setBurger] = useState(true);
+
+  return (
+    <nav className="flex flex-col justify-center items-center h-[55px] relative">
+      <div className="flex flex-row-reverse justify-around items-center w-screen mt-3 lg:mt-0">
+        <div>
+          {burger ? (
+            <GiHamburgerMenu
+              onClick={() => setBurger(!burger)}
+              className="burger-icons"
+            />
+          ) : (
+            <VscChromeClose
+              onClick={() => setBurger(!burger)}
+              className="burger-icons"
+            />
+          )}
+        </div>
+        <div className="inline-block lg:hidden lg:mr-[6rem] hover:scale-110 ease-in-out transition-all">
+          <Link
+            to="/question+"
+            className="p-[7px] rounded-md border-[1px] border-cusOrange lg:p-2 "
+          >
+            <span className="text-sm text-cusOrange lg:text-base">
+              Ask Question+
+            </span>
+          </Link>
+        </div>
+        <SiTailwindcss className="block text-[2rem] lg:hidden " />
+      </div>
+
+      <div
+        onClick={() => setBurger(true)}
+        aria-hidden="true"
+        className={`ml-2 w-screen top-[4rem] lg:ml-0 lg:flex lg:justify-between lg:items-center lg:h-[4rem] lg:top-[1px] absolute ${
+          burger ? 'hidden' : null
+        }`}
+      >
+        <div>
+          <ul className="inline-block mt-4 text-2xl lg:mt-0 lg:text-base lg:flex lg:items-center">
+            <SiTailwindcss className="hidden mb-3 lg:mb-0 mx-10 text-[2rem] fill-current lg:block" />
+            <li className="mb-3 lg:mb-0 lg:pr-3 lg:border-r-[1px] lg:border-cusOrange hover:text-cusOrange transition-all ease-in-out">
+              <Link to="/">Home</Link>
```

Comment:
Let's bring back the routes.js file and have this be there and import them here. 
It will be like 
```js
<Link to={HOME_ROUTE}>Home</Link>
```